app/main.py

```diff
@@ -5,8 +5,9 @@ def get_human_age(cat_age: int, dog_age: int) -> list:
 
 
 def convert_to_human(
-        animal_age: int, first_year: int, second_year: int, each_year: int
-) -> int:
+        animal_age: int,
+        first_year: int, second_year: int,
```

Comment:
The use of the division operator `/` in the parameter list is incorrect. This is likely intended to be a separator for positional-only parameters, but it is not correctly implemented. Please review the syntax for defining positional-only parameters in Python.